money/currency.py

```diff
@@ -575,13 +575,13 @@ class CurrencyHelper:
             'display_name': 'Forint',
             'numeric_code': 348,
             'default_fraction_digits': 2,
-            'sub_unit': 100,
```

Comment:
now for this one, the two lists disagree but Wikipedia says the sub unit is defunct, so maybe Facebook's is more accurate here

Reply:
Was in Hungary this summer & did not ever encounter any denomination smaller than a Forint. Inflation is crazy there so it makes sense that the sub unit would be defunct. Based on those [Facebook docs](https://developers.facebook.com/docs/marketing-api/currencies/#example-offset-1) it looks like 1 indicates that there is no sub unit allowed, so it seems they actually agree here!